Model/Reclaim.php

```diff
@@ -284,17 +284,24 @@ public function _storeFilter($storeId)
         return $storeIdFilter;
     }
 
-    public function _getImages($product){
+    public function _getImages($product)
+    {
         $images = $product->getMediaGalleryImages();
         $image_array = array();
+        
         foreach($images as $image) {
-            $image_url = $image->getUrl();
-            if ($image_url){
-                $image_array[] = $image_url;
-            }
+            $image_array[] = $this->handleMediaURL($image);
         }
         return $image_array;
     }
+    public function handleMediaURL($image)
+    {
+        $custom_media_url = $this->_klaviyoHelper->getCustomMediaURL();
+        if ($custom_media_url){
```

Comment:
can we also check if image->getFile() is not null?

Reply:
everytime an image is uploaded, the file is stored.  it will always be there, it's part of magento core